Clients/GUI/Scripts/BrokerManager.cs

```diff
@@ -0,0 +1,33 @@
+﻿using DTO;
+using DTO.MarketBrokerObjects;
+using Kernel;
+using Kernel.Enums;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace GUI.Scripts
+{
+    public static class BrokerManager
+    {
+        public static async Task<IEnumerable<Instrument>> GetInstruments(
+                BrokerType broker,
+                string token,
+                InstrumentType instrument,
+                int depth = 10)
+        {
+            const string url = "https://localhost:5009/operations/instruments/get";
+
+            var queryParams = new Dictionary<string, string>
+            {
+                { "bank", broker.ToString() },
+                { "token", token },
+                { "depth", depth.ToString() },
+                { "instrument", instrument.ToString() }
+            };
+
+            var client = new RestClient<object, IEnumerable<Instrument>>(url, RestRequestType.GET, queryParams: queryParams);
+
+            return await client.Execute();
+        }
+    }
+}
```

Comment:
Perhaps we should leave an empty line at the end of the file (this problem is in almost every file of the extraction request).

Reply:
Convonce me why I have to spend time doing it for each existing file or suggest some extension to do it in one click